br/pkg/lightning/mydump/parser_generated.go

```diff
@@ -336,7 +336,6 @@ func (parser *ChunkParser) lex() (token, []byte, error) {
 					goto _out
 				}
 			}
-			goto st21
```

Comment:
can you make whatever linter *ignore* this file instead of fixing it? this file is auto-generated and so shouldn't be changed.